test/unit/server.spec.js

```diff
@@ -99,18 +106,27 @@ describe('server', () => {
       close: sinon.spy((callback) => callback && callback())
     }
 
-    sinon.stub(server._injector, 'get')
-      .withArgs('webServer').returns(mockWebServer)
-      .withArgs('socketServer').returns(mockSocketServer)
+    sinon
+      .stub(server._injector, 'get')
+      .withArgs('webServer')
+      .returns(mockWebServer)
```

Comment:
minor: I would put `returns()` on the same line as the corresponding `withArgs()`